### PR TITLE
Page setup

### PR DESCRIPTION
**Pull request by 3rkeen**
Planned and positioned various parts of the project. Project lay-out includes positiong the navigation area, the main body containing the embeded video, the aside containing the thumbnail images and title, the comment section and footer.

### CODE REVIEW

**Comment by addod19** [comment]
## Hi Team,
Your project will be marked as invalid because of the following reason:
- The project has no descriptive PR. PR must have a title and a short description.

- The project has no professional** README**  file

**Comment by Zubenna** [comment]
Created layout for the webpage

**Review by mosesogwo — CHANGES_REQUESTED** [review]
**HI @3rkeen, @Zubenna. Good job on your first project:clap:. Let's now take care of the following :arrow_forward:**

1. Reduce the font-size of video titles in the recommended videos side bar to enable the rest of the information fit within the video height.
**Original:**
image

**Yours:**
image

2. Position the subscribe button properly

3. Remove all inline stylings from your html

4. You have a single warning from the html validation. You could add a header to that section and set it s display property to none.

5. You should generate a live link for your page at https://raw.githack.com and include it in your readme.

6. Author 1 Information in the readme have no links. There should be links to the various platforms identified.

7. Use hyperlinks for links in your readme to keep it neat. Consider using the Microverse template provided [here](https://github.com/microverseinc/readme-template)

_**Kindly address the above and submit another code review request. Note that you don't have to open another pull request.**_

**Cheers:stars:**

File: index.html [hunk 1]
```diff
@@ -0,0 +1,330 @@
...
+                >
+              </div>
+            </div>
+       </nav>
+    </header>
+    <main class="video video-grid">
+        <iframe src="https://www.youtube.com/embed/zrJtFy51fRo" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+          <p>Luis Fonsi - Despacito ft. Daddy Yankee</p>
+          <div class="section-icon">
+            <p style="font-size: small;">6,636,870,358 views • Jan 13, 2017</p>
```
Comment (mosesogwo) [inline index.html]: Move this style into the css

**Review by mosesogwo — CHANGES_REQUESTED** [review]
**Hello, we haven't made all the changes from the previous review yet:**

1. The subscribe button is not properly placed. It should fall in line with the author avatar.
**Yours:** 
image

**Original:**
image

2. You still have one inline style in your html. See line 51 of your index.html

_**Please fix them and submit another code review request. You're almost there :muscle:**_

**Review by mosesogwo — APPROVED** [review]
**You have done well implementing the changes:+1: Your project is now approved!:heavy_check_mark:**

You can now merge to master.

**Good luck in your next project:stars:**

**Pull request merged**